go/test/endtoend/vtorc/readtopologyinstance/main_test.go

```diff
@@ -93,7 +93,7 @@ func TestReadTopologyInstanceBufferable(t *testing.T) {
 	assert.Contains(t, primaryInstance.InstanceAlias, "zone1")
 	assert.NotEqual(t, 0, primaryInstance.ServerID)
 	assert.Greater(t, len(primaryInstance.ServerUUID), 10)
-	assert.Contains(t, primaryInstance.Version, "5.7")
+	assert.Contains(t, primaryInstance.Version, "8.0")
```

Comment:
We can use `assert.Regexp(t, `[58]\.[07].*`, replicaInstance.Version)` instead. This will make this test work with both 5.7 and 8.0